susemanager-utils/testing/docker/scripts/schema_idempotency_test_pgsql.py

```diff
@@ -220,7 +220,7 @@ def main():
         sys.exit(0)
     if args.pr_file:
         raise RuntimeError(
-            "Dumps %s and %s are different! One or more scripts from the PR are not idempotent!")
+            "Dumps %s and %s are different! One or more scripts from the PR are not idempotent!" % (initial_dump, migrated_dump, args.version))
```

Comment:
Looks like there is 3 arguments, but only 2 `%s`, is this expected? Or should it be just `initial_dump` and `migrated_dump`?

Reply:
OUCH! Wrong copy paste. You are right.